### PR TITLE
Feat: display org data on org page #162

### PR DESCRIPTION
**Pull request by kevin-hashimoto**
- feat: updates the api route to fetch org data from the db
  - new schema made specifically for the route with a more scalable and modern approach. other org schemas left untouched in case it messes with what is already working.
- feat: replaces org dummy data with real data
- feat: `constructAddress` util function that takes in an address object ( same format as what is returned from backend ) and spits out a formatted full address

### CODE REVIEW

File: backend/lcfs/web/api/organization/views.py [hunk 1]
```diff
@@ -5,28 +5,39 @@
...
+    GetOrganizationResponse,
+)
 from lcfs.web.core.decorators import roles_required
 
 logger = getLogger("organization")
 router = APIRouter()
 get_async_db = dependencies.get_async_db_session
 
+
 # TODO: Implement permission check for this route to ensure that
```
Comment (AlexZorkin) [inline backend/lcfs/web/api/organization/views.py:31]: plz remove this comment, no longer needed.

File: backend/lcfs/web/api/organization/views.py [hunk 2]
```diff
@@ -5,28 +5,39 @@
...
 # only authorized users can access it.
-@router.post("/", response_model=OrganizationSchema, status_code=status.HTTP_201_CREATED)
+@router.post(
+    "/", response_model=OrganizationSchema, status_code=status.HTTP_201_CREATED
+)
 @roles_required("Government", "Adminstrator")
 async def create_organization(
```
Comment (AlexZorkin) [inline backend/lcfs/web/api/organization/views.py:37]: I'm getting a duplicate key error when trying to create a new organization. Are you able to create in your local environment?

File: backend/lcfs/web/api/organization/views.py [hunk 3]
```diff
@@ -62,17 +72,57 @@ async def create_organization(
...
+
+        return org
+
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Internal Server Error: {str(e)}")
+
+
 @router.put("/{organization_id}", response_model=OrganizationSchema)
-async def update_organization(organization_id: int, organization_data: OrganizationUpdateSchema, db: AsyncSession = Depends(get_async_db)):
+async def update_organization(
```
Comment (AlexZorkin) [inline backend/lcfs/web/api/organization/views.py:109]: We don't have an edit view for org yet, so just a note that we'll have to test this again once the view is created.

**Review by AlexZorkin — APPROVED** [review]
Looks good, just a few comments to address.

File: backend/lcfs/web/api/organization/views.py (hunk 3 above)
Comment (kevin-hashimoto) [reply backend/lcfs/web/api/organization/views.py:109]: I think prashanth is working on the edit org side of things in a separate ticket.

File: backend/lcfs/web/api/organization/views.py (hunk 2 above)
Comment (kevin-hashimoto) [reply backend/lcfs/web/api/organization/views.py:37]: I've been using the seeded data to test so I havnt run in to this issue. Ill test it out on my local

Comment (kevin-hashimoto) [reply backend/lcfs/web/api/organization/views.py:35]: this should be fixed now. will need to delete and seed the db again. the sequence was off when seeding so it caused dupe keys

**Pull request merged**